docs/modules/operation/pages/flows/basic.adoc

```diff
@@ -0,0 +1,142 @@
+[source, xml]
+----
+# ElasticSearch persistence configuration
+elasticUrl = http://10.10.3.218:9200 <1>
+connTimeout = 30000
+readTimeout = 300000
+settings.index.number_of_replicas = 0
+settings.index.number_of_shards=1
+settings.index.refresh_interval=10s
+elasticIndexStrategy=daily
+----
+<1> Replace with comma-separated list of Elasticsearch nodes.
+
+See <<elasticsearch/introduction.adoc#ga-elasticsearch-integration-configuration, General Elasticsearch Configuration>> for a complete set of options.
+
+== Enable protocols
+
+Update `$\{OPENNMS_HOME}/etc/telemetryd-configuration.xml` to enable one or more of the protocols you want to handle.
+
+This example enables the NetFlow v5 protocol.
+Use the same process for any of the other flow-related protocols.
+
+[source, xml]
+----
+<listener name="Netflow-5-UDP-8877" class-name="org.opennms.netmgt.telemetry.listeners.UdpListener" enabled="true">
+    <parameter key="port" value="8877"/>
+
+    <parser name="Netflow-5-Parser" class-name="org.opennms.netmgt.telemetry.protocols.netflow.parser.Netflow5UdpParser" queue="Netflow-5" />
+</listener>
+
+<queue name="Netflow-5">
+    <adapter name="Netflow-5-Adapter" class-name="org.opennms.netmgt.telemetry.protocols.netflow.adapter.netflow5.Netflow5Adapter" enabled="true">
+    </adapter>
+</queue>
+----
+
+Send a `reloadDaemonConfig` event via the CLI to apply the changes without restarting:
+
+[source, console]
+----
+$\{OPENNMS_HOME}/bin/send-event.pl -p 'daemonName Telemetryd' uei.opennms.org/internal/reloadDaemonConfig
+----
+
+This opens a UDP socket bound to `0.0.0.0:8877` to which NetFlow v5 messages are forwarded.
+
+=== Multi-port listener
+
+If you are monitoring multiple flow protocols, you normally need to set up a flow listener for each one, on its own UDP port.
+
+By default, {page-component-title} enables a multi-port listener option, which monitors multiple protocols on a single UDP port (9999).
+If desired, edit `$\{OPENNMS_HOME}/etc/telemetryd-configuration.xml` to change the port number or add/remove protocols.
+
+IMPORTANT: Make sure any ports you configure for receiving flow data are added to your firewall allow list.
+
+== Enable flows on your device(s)
+
+Configure your devices to send flows.
+Refer to the manufacturer's documentation.
+You will need to set up the flow receiver, which is OpenNMS {page-component-title}, and enable sending flows per interface on the firewall.
+
+== Link the web UI to Helm
+
+To access flow-related graphs from the {page-component-title} web interface, you must configure a link to your instance of OpenNMS Helm.
+
+.Connect to Karaf shell
+[source, console]
+----
+ssh -p 8101 admin@localhost
+----
+
+.Configure Helm settings within Karaf
+[source, karaf]
+----
+config:edit org.opennms.netmgt.flows.rest
+config:property-set flowGraphUrl 'http://grafana:3000/dashboard/flows?node=$nodeId&interface=$ifIndex'
+config:update
+----
+
+NOTE: This URL can optionally point to other tools as well.
```

Comment:
I am wondering which other tools are meant here. Maybe add 1-2 examples?

Reply:
Great idea. @mmahacek or @indigo423 any thoughts on other tools?

Reply:
I see the potential, but I don't know of any other tools that are feasible at this time.